script/zmb

```diff
@@ -367,8 +367,11 @@ sub cmd_get_test_history {
         },
     );
 
-    if ( $opt_nameserver ) {
-        $params{frontend_params}{nameservers} = json_tern( $opt_nameserver );
+    if ( $opt_filter ) {
+        unless ( $opt_filter =~ /^(?:all|delegated|undelegated)$/ ) {
+            die 'Illegal filter value. Expects "all", "delegated" or "undelegated" ';
```

Comment:
Is the trailing space intentional?

Reply:
Yes, since there is no LF `die` will add a reference to the place in the code, but now I see that it also adds a space before that. I can remove the trailing space, but I guess that it really does not matter.

Reply:
With an extra space we get a clearer separation between the message and the line reference. I don't believe I've seen that before but I think it's a nice feature.

Reply:
Yes, you are right.
```
# ~matsd/zmb --verbose get_test_history --domain iis.se --filter deleg | jq .
Illegal filter value. Expects "all", "delegated" or "undelegated"  at /home/matsd/zmb line 372.
```